version.rb

```diff
@@ -0,0 +1,3 @@
+load('VERSION')
+
+WORKFLOW_VERSION="#{MAJOR}.#{MINOR}.#{PATCH}.#{PRE_RELEASE_IDENTIFIER}"
```

Comment:
Doesn't CocoaPods expect prerelease versions to look like this: `1.0.0-alpha1`?

This also needs to account for when we drop the prerelease identifier for 1.0 (though we could deal with that then).

Reply:
Yes, that last `.` should be `-`. Will fix.
Yes, I'll fix for the `PRE_RELEASE_IDENTIFIER`.